src/components/tile/tile.stories.tsx

```diff
@@ -167,6 +167,23 @@ export const WithTileFooter: ComponentStory<typeof Tile> = () => (
         <TileFooter p={2} variant="black" />
       </TileContent>
     </Tile>
+    <Box my={3} />
```

Comment:
Comment (non-blocking): `Tile` supports margin so you could have passed `mt={3}` here but it's not a huge issue for me how you've done it

Reply:
Yeah I agree, this is a really dirty way of doing it and putting margin on the Tile is better - I only did it this way because it's already done on the same story (I'd just managed to lose that part when messing with the stories). We can tidy this up later, but since the ticket is urgent and there's nothing here that affects consumers, let's let it pass for now.